setup.py

```diff
@@ -110,7 +110,9 @@ def run(self):
       cmdclass=cmdclass,
       # rosinstall dependency to be kept in order not to break ros hydro install instructions
       install_requires=install_requires,
-      scripts=["scripts/wstool"],
+      entry_points = {
+        'console_scripts': ['wstool=wstool.wstool_cli:main']
```

Comment:
Wow, is this a new feature? :dancer:

Reply:
This console_scripts seems to be a feature for a while (https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html), but since wstool mostly was used in the system supporting shebang so it probably doesn't matter to generate the console program stub.

However on Windows, it doesn't magically recognize it is a python script so people usually need to do "python wstool" to run it. With this console_scripts help, now on Windows you can also do "wstool". :)

Reply:
:raised_hands: